Lib/test/test_sqlite3/test_cli.py

```diff
@@ -158,6 +160,14 @@ def test_interact_on_disk_file(self):
         out, _ = self.run_cli(TESTFN, commands=("SELECT count(t) FROM t;",))
         self.assertIn("(0,)\n", out)
 
+    def test_color(self):
+        with unittest.mock.patch("_colorize.can_colorize", return_value=True):
```

Comment:
It is a little weird to fight the class decorator here, but I guess we're fine for such a simple script.